src/main/java/seedu/us/among/model/endpoint/Address.java

```diff
@@ -16,7 +16,7 @@
 public class Address {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Addresses should be in the form of a url. E.g. https://localhost:3000";
+            "Addresses should be in the form of a valid url. E.g. https://localhost:3000";
```

Comment:
Actually would it be better to use an actual url that works here? Just in case people actually paste this url to try and because it doesn't work, they flag it as a bug ._.

Reply:
done